onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -1511,16 +1511,15 @@ def aten_einsum(
     raise NotImplementedError()
 
 
+@torch_op("aten::embedding")
 def aten_embedding(
-    weight: TensorType,
-    indices: TensorType,
-    padding_idx: int = -1,
-    scale_grad_by_freq: bool = False,
-    sparse: bool = False,
-) -> TensorType:
+    weight: TTensor,
+    indices: TTensor,
+    **kwargs,# pylint: disable=unused-argument
```

Comment:
```suggestion
    **_,
```

Don’t know if ** works in onnxscript, but if it does:

Reply:
Do we need to change behavior based on padding_idx?

Reply:
According to current implementation, it will only impact Training. I think ONNX Script won't know if current export is under Training mode, so the warning about possible wrong training result should be thrown by exporter, not onnx-script.

Thoughts?

Reply:
I think we should also support any training behaviors when possible.